rerun_py/src/python_bridge.rs

```diff
@@ -544,11 +545,20 @@ fn connect(
 
     let flush_timeout = flush_timeout_sec.map(std::time::Duration::from_secs_f32);
 
+    let ready_opts = BlueprintReadyOpts {
+        make_active: false,
+        make_default: true,
+    };
```

Comment:
we can shorten a lot of code in this file with a few constants

```suggestion
    let ready_opts = BlueprintReadyOpts::MAKE_DEFAULT;
```